src/env/environment_utils.c

```diff
@@ -0,0 +1,14 @@
+#include <env/environment.h>
```

Comment:
Do I understand correctly that `key_len` minusses the addresses of the two positions? So that in ex:
for: "USER=HALLO"
delimiter_position = adddress of the '='?:
&key_value_str[0] = address of the 'U'?;
So the difference will be 4?
Cause if so this is fucking genius!!!!! :D

Reply:
Exactly haha
It's cool right?
If we calculate the difference between addresses we get the len 😃